internal/db/migrations.go

```diff
@@ -38,17 +41,38 @@ func Migrate() {
 	mlog := log.Logger.With().Bool("migration", true).Logger()
```

Comment:
All logs related to migrations go into zerolog with migration=true flag so we can easily filter them.